pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -68,6 +68,7 @@ type PostgresSpec struct {
 	Tolerations           []v1.Toleration             `json:"tolerations,omitempty"`
 	Sidecars              []Sidecar                   `json:"sidecars,omitempty"`
 	InitContainers        []v1.Container              `json:"initContainers,omitempty"`
+	ReadOnlyRootFilesystem *bool                       `json:"readOnlyRootFilesystem,omitempty"`
```

Comment:
looks like go.fmt did not run here. All other lines should normally adapt to the new option. Remove also the two spaces introduced above.